src/test/java/uk/gov/hmcts/reform/sscs/service/LanguageServiceTest.java

```diff
@@ -14,110 +14,6 @@ public void setup() throws IOException {
-        Assert.assertNotNull(intepreterDescription);
-        Assert.assertEquals("a sign language interpreter (Manual Alphabet)", intepreterDescription);
-    }
-
-    @Test
-    public void givenASignLanguageMakatonKey_ThenExtractTheInterpreterDescription() {
-        String intepreterDescription = service.getInterpreterDescriptionForLanguageKey("signMakaton");
-        Assert.assertNotNull(intepreterDescription);
-        Assert.assertEquals("a sign language interpreter (Makaton)", intepreterDescription);
-    }
-
-    @Test
-    public void givenASignLanguageLipspeakerKey_ThenExtractTheInterpreterDescription() {
-        String intepreterDescription = service.getInterpreterDescriptionForLanguageKey("signLipSpeaker");
-        Assert.assertNotNull(intepreterDescription);
-        Assert.assertEquals("a sign language interpreter (Lip Speaker)", intepreterDescription);
-    }
-
-    @Test
-    public void givenASignLanguageInternationalSignKey_ThenExtractTheInterpreterDescription() {
-        String intepreterDescription = service.getInterpreterDescriptionForLanguageKey("signInternationalSign");
-        Assert.assertNotNull(intepreterDescription);
-        Assert.assertEquals("a sign language interpreter (International Sign)", intepreterDescription);
-    }
-
-    @Test
-    public void givenASignLanguageDeafBlindKey_ThenExtractTheInterpreterDescription() {
-        String intepreterDescription = service.getInterpreterDescriptionForLanguageKey("signDeafBlind");
-        Assert.assertNotNull(intepreterDescription);
-        Assert.assertEquals("a sign language interpreter (Deaf Blind ‑ Hands on)", intepreterDescription);
-    }
-
-    @Test
-    public void givenASignLanguageBslKey_ThenExtractTheInterpreterDescription() {
-        String intepreterDescription = service.getInterpreterDescriptionForLanguageKey("signBsl");
-        Assert.assertNotNull(intepreterDescription);
-        Assert.assertEquals("a sign language interpreter (BSL)", intepreterDescription);
-    }
-
-    @Test
-    public void givenASignLanguageAmericanSignKey_ThenExtractTheInterpreterDescription() {
-        String intepreterDescription = service.getInterpreterDescriptionForLanguageKey("signAmericanSign");
-        Assert.assertNotNull(intepreterDescription);
-        Assert.assertEquals("a sign language interpreter (American Sign)", intepreterDescription);
-    }
-
-    @Test
-    public void givenAValidNonSignLanguageKey_ThenExtractTheInterpreterDescription() {
-        String intepreterDescription = service.getInterpreterDescriptionForLanguageKey("senegalFrench");
-        Assert.assertNotNull(intepreterDescription);
-        Assert.assertEquals("an interpreter in Senegal (French) Olof Dialect", intepreterDescription);
-    }
-
-    @Test
-    public void givenAnInvalidLanguageKey_ThenReturnNullInterpreterDescription() {
-        String languageName = service.getInterpreterDescriptionForLanguageKey("something");
-        Assert.assertNull(languageName);
-    }
```

Comment:
You should not remove these tests, just move them to AdjournCasePreviewServiceTest.java and refactor them to test where this functionality was moved to